athena.go

```diff
@@ -64,6 +64,8 @@ func main() {
 	case "fatal":
 		logger.CurrentLevel = logger.Fatal
 	}
+	logger.LogStdOut = config.LogStd
+	logger.LogFile = config.LogFile
```

Comment:
Instead of using two different config options, it would be better to have a single config option for defining the logging methods, perhaps using a string array - which would allow for (potentially) more logging methods to be added without requiring additional config options.